content/get-started/transactions.mdx

```diff
@@ -3,6 +3,20 @@ title: Transacting
 seo-title: Sending Transactions
 ---
 
+## Check your balance
+
+You can check your balance at any time by running the following command:
+
+```sh
+ironfish wallet:balance
+```
+
+Your wallet must be synced with the block chain or it will be wrong. You can ask for a balance explanation by running
```

Comment:
nit - we seem to use `blockchain` as one word elsewhere
```suggestion
Your wallet must be synced with the blockchain or it will be wrong. You can ask for a balance explanation by running
```